components/About/About.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const About = () => {
+  return (
+    <div className='container'>
```

Comment:
`<div className='about container'>`

Reply:
I´going to add this class, I think fix it.

[class$="__container"]{
  max-width: 140rem;
  width: 90%;
  margin: 0 auto;
}

Reply:
I cant see this change?

Reply:
I cant see this change?